src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -451,14 +451,16 @@ class PeopleReport extends Component {
             </h4>
             <p>End Date</p>
           </div>
-          {!this.state.isBioPosted ? (
+          {this.state.bioStatus ? (
             <div>
-              <h4>Bio {this.state.isBioPosted ? 'posted' : 'requested'}</h4>{' '}
+              <h5>Bio {this.state.bioStatus === "default" ? "not requested" : this.state.bioStatus}</h5>{' '}
```

Comment:
Old accounts that haven't been modified have their bioPosted (which gets set to this.state.bioStatus) as true or false. 
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/98581e72-7900-48e6-aedf-1adc6babca67)

Reply:
I made changes to the old accounts that are set to true and false. I have manually adjusted their document values to change from false -> "requested", true -> "posted".